src/test/resources/org/openeo/geodiasFeatureSnippet.json

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
minor, but is it intentional to call this 'geodias', instead of 'creodias'?

Reply:
Typo! Fixed it now